tablo/geom_utils.py

```diff
@@ -437,4 +437,196 @@ def extract_significant_digits(number):
     if is_negative:
         rounded = 0 - rounded
 
-    return rounded
\ No newline at end of file
+    return rounded
+
+
+def wkt_to_esri_feature(wkt):
+    geom_type = wkt[:wkt.find('(')]
+
+    def _geom_repl():
+        bracket_multiplier = 1
+        if geom_type.find('LINESTRING') > -1:
+            bracket_multiplier = 2
+        return json.loads(re.sub(
+            WKT_GEOM_2D_REGEX,
+            lambda m: '[{}]'.format(m.group().replace(' ', ',')),
+            wkt.replace(geom_type, '')
+        ).replace('(', '[' * bracket_multiplier).replace(')', ']' * bracket_multiplier))
+
+    if geom_type == 'POINT' or geom_type == 'MULTIPOINT':
+        match = WKT_POINT_REGEX.match(wkt)
+        if not match:
+            raise ValueError('Invalid Point Geometry: {0}'.format(wkt))
+        x, y = (float(x) for x in match.groups())
+        return {'x': x, 'y': y}
+    elif geom_type == 'LINESTRING' or geom_type == 'MULTILINESTRING':
+        return {'paths': _geom_repl(wkt, geom_type)}
+    elif geom_type == 'POLYGON' or geom_type == 'MULTIPOLYGON':
+        return {'rings': _geom_repl(wkt, geom_type)}
+
+
+def esri_feature_to_ewkt(feature, geom_type):
+    srid = 'srid={};'.format(feature.get('spatialReference', {}).get('wkid', 3857))
```

Comment:
It might be clearer to use an explicit index in the format string: `'srid={0};'.format(...)`

...or this might be even more explicit: `'srid={wkid};'.format(wkid=...)`